Library/Homebrew/cmd/update.sh

```diff
@@ -210,11 +210,7 @@ merge_or_rebase() {
 
   if [[ "${DIR}" == "${HOMEBREW_REPOSITORY}" && -n "${HOMEBREW_UPDATE_TO_TAG}" ]]
   then
-    UPSTREAM_TAG="$(
-      git tag --list |
-        sort --field-separator=. --key=1,1nr -k 2,2nr -k 3,3nr |
-        grep --max-count=1 '^[0-9]*\.[0-9]*\.[0-9]*$'
-    )"
+    UPSTREAM_TAG="$(git tag --list --sort=-version:refname | grep -m 1 '^[0-9]*\.[0-9]*\.[0-9]*$')"
```

Comment:
Can you preserve the previous multi-line formatting here please?

Reply:
Done.